src/components/DeleteButtonEdge.tsx

```diff
@@ -5,13 +5,15 @@ import { useIntegrationJsonStore, useNestedStepsStore, useVisualizationStore } f
 import { IVizStepNode } from '@kaoto/types';
 import { Button, Popover, Tooltip } from '@patternfly/react-core';
 import { MinusIcon } from '@patternfly/react-icons';
-import { getBezierPath, Position, useReactFlow } from 'reactflow';
+import { ReactNode } from 'react';
+import { EdgeText, getBezierPath, Position, useReactFlow } from 'reactflow';
 
 const foreignObjectSize = 40;
 
 export interface IDeleteButtonEdge {
   data?: any;
   id: string;
+  label?: string | ReactNode;
```

Comment:
If I'm not mistaken, the `ReactNode` type already include `string` in its definition, so maybe is not needed to explicitly pass it here.

```typescript
    type ReactNode = ReactElement | string | number | ReactFragment | ReactPortal | boolean | null | undefined;
```

```suggestion
  label?: ReactNode;
```

Reply:
ugh I missed this one, and right, just `ReactNode` was enough - will follow up. I blame React Flow as this is just a stolen one from its source :grin: